phi/model/content.py

```diff
@@ -14,5 +14,6 @@ class Video(BaseModel):
 class Image(BaseModel):
     id: str
     url: str
+    alt_text: Optional[str] = None
```

Comment:
Thoughts on using a `meta_data` param here instead?

Reply:
Hmm I think `metadata` would be a good idea on top of alt_text. Alt_text is a pretty standard image parameter, right?